ceph_deploy/hosts/centos/install.py

```diff
@@ -198,11 +200,8 @@ def repo_install(distro, reponame, baseurl, gpgkey, **kw):
 
     # set the right priority
     if kw.get('priority'):
-        distro.packager.install_priorities_plugin()
-        logger.warning(
-            'ensuring that {repo_path} contains a high priority'.format(
-                repo_path=repo_path)
-        )
+        if isinstance(distro.packager, Yum):
```

Comment:
it seems that it would be good to have a way to tell if this is yum besides using `isinstance`. I know that we have `executable` and you could do: `if distro.packager.executable == 'yum':` but it seems that for Apt we are using a list.

I don't think that a list is needed in `Apt` and could be normalized to `apt-get` and then this would work.

Otherwise, you may want to add something like `.name` ? 

Either way, I see this being used often and might be a good idea.